lib/rules/no-invalid-attribute-name.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @author Doug Wade <douglas.b.wade@gmail.com>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+const xnv = require('xml-name-validator')
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require valid attribute names',
+      categories: undefined,
+      url: 'https://eslint.vuejs.org/rules/no-invalid-attribute-name.html'
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      attribute: 'Attribute name {{name}} is not valid.',
+      directive: 'Directive name {{name}} is not valid.'
```

Comment:
I think it's `'v-bind' argument`, not `Directive name`.

Reply:
When use v-bind on native HTML element, it translates to an attribute, so I reconsidered that it would be better to use the same message as `attribute`.
What do you think?